docs/docs/configuration/materializations/hive-ha.md

```diff
@@ -43,3 +45,12 @@ By default, the materialization keeps the last 4 table versions. You can change
 
 - When swapping from a table with partitions to a table without (and the other way around), there could be a little downtime. In case high performance is needed, consider bucketing instead of partitions
 - By default, Glue "duplicate" the versions internally, so the last 2 versions of a table point to the same location. Therefore it's recommended to use `versions_to_keep` >= 4, as this will avoid to have the older location removed.
+
+</VersionBlock>
+
+<VersionBlock firstVersion="1.5">
+
+This materialization has been removed and is now part of the `table` materialization, configurable with `ha=true`.
+You can find more about it in the table materialization page about [Hive tables](./hive.md).
+
+</VersionBlock>
```

Comment:
Another option would be to just start hiding the page in 1.5 with `lastVersion` in `dbt-athena-versions.js`... Using search you will still end up on the hive page.

👉 https://github.com/dbt-labs/docs.getdbt.com/blob/d4817d192ab2629945052b2832318de5420c9a44/contributing/single-sourcing-content.md#properties-for-versioning-an-entire-page

Reply:
I did that, but it does not work well, I don't know if I am missing something 🤔 

<img width="738" alt="Screenshot 2023-05-12 at 11 28 27" src="https://github.com/dbt-athena/dbt-athena.github.io/assets/9251353/da6882b4-e970-4bbb-b2a8-e948a89b0fd6">

Reply:
Hmm so it disappears in the sidebar on 1.5 but if you bookmarked the url (or visit it via google) and switch to v1.5, it still is visible 🤔

Reply:
That seems so. What would you recommend ?

Reply:
I looked through the code and it can only show a block for versions that are new https://github.com/dbt-athena/dbt-athena.github.io/blob/main/src/theme/DocPage/Layout/Main/index.js#L97-L104

We could add a check if the page is no longer in use. Either as a global message in `index.js` or a unique message on the page itself.

- ✅ Setting `lastVersion` will hide it in sidebar on v1.5
- ⌛ If you accidentally visit the url anyway on v1.5, we could show a message on the page specific for this feature to direct them to the hive page instead?

```markdown
<VersionBlock firstVersion="1.5">
:::caution
This feature changed in version 1.5. 
Visit the [hive materialization](docs/configuration/materialization/hive) page for more information, or browse the [migration guide](docs/migration/UPGRADE-1.5.md) for more details.
:::
</VersionBlock>
```

Reply:
I did a check and seems working as expected, if someone bookmark the ulr, then switch to 1.5 that appear, seems indeed what is needed, great work folks 💯